generators/server/files.js

```diff
@@ -613,6 +613,36 @@ const serverFiles = {
         },
       ],
     },
+    {
+      condition: generator => generator.applicationType === 'gateway' && generator.serviceDiscoveryType,
+      path: SERVER_MAIN_SRC_DIR,
+      templates: [
+        {
+          file: 'package/config/GatewaySwaggerResourcesProvider.java',
+          renameTo: generator => `${generator.javaDir}config/GatewaySwaggerResourcesProvider.java`,
+        },
+      ],
+    },
+    {
+      condition: generator => generator.applicationType === 'gateway' && generator.serviceDiscoveryType,
```

Comment:
as it's the same condition than above, maybe merge it ?

Reply:
done :)